fix a name of method in tests/

```
tests/command_download_atcoder.py:120: error: Name 'test_call_download_atcoder_abc114_c' already defined on line 102
```

https://github.com/kmyk/online-judge-tools/blob/51691b7d0d7e375eb24b4c804aaeb6fe36cc80fb/tests/command_download_atcoder.py#L97

--- a/tests/command_download_atcoder.py
+++ b/tests/command_download_atcoder.py
@@ -117,7 +117,7 @@ class DownloadAtCoderTest(unittest.TestCase):
             ],
             type='json')
 
-    def test_call_download_atcoder_abc114_c(self):
+    def test_call_download_atcoder_abc003_4(self):
         self.snippet_call_download(
             'https://atcoder.jp/contests/abc003/tasks/abc003_4', [
                 {

